examples/stocks/lib/stock_home.dart

```diff
@@ -59,24 +92,31 @@ class StockHomeState extends State<StockHome> {
     });
   }
 
-  bool _autorefresh = false;
-  void _handleAutorefreshChanged(bool value) {
-    setState(() {
-      _autorefresh = value;
-    });
-  }
-
   void _handleStockModeChange(StockMode value) {
     if (config.updater != null)
       config.updater(config.configuration.copyWith(stockMode: value));
   }
 
-  void _handleMenuShow() {
-    showStockMenu(
-      context: context,
-      autorefresh: _autorefresh,
-      onAutorefreshChanged: _handleAutorefreshChanged
-    );
+  void _handleStockMenu(BuildContext context, _StockMenuItem value) {
+    switch(value) {
+      case _StockMenuItem.autorefresh:
+        setState(() {
+          _autorefresh = !_autorefresh;
+        });
+        break;
+      case _StockMenuItem.refresh:
+        showDialog(
+          context: context,
+          child: new _NotImplementedDialog()
```

Comment:
Right, that's why I would ahve inlined it. Giving it a class makes it look less hacky. ;-)

It's fine either way.